docs/working-copy.md

```diff
@@ -84,3 +84,12 @@ A's working copy will become stale. By "stale", we mean that the files in the
 working copy don't match the desired commit indicated by the `@` symbol in
 `jj log`. When that happens, use `jj workspace update-stale` to update the files
 in the working copy.
+
+If that doesn't work for some reason, you probably found a bug in `jj`. Check if
```

Comment:
I don't think this paragraph is specific to stale working copy, and can be removed.

Reply:
Thinking out loud, the things I wanted to communicate here were:

- This is an area where `jj` has known bugs.
- Please consider filing a bug *before* recovering.
- I don't think we put the instructions for archiving the repository anywhere else.

For the first of this, it would probably be better to say it explicitly and link to the bugs.

I'm not sure how to better communicate the second one.

The third one might belong somewhere else.

Let me know if you have any thoughts. In any case, I'll redo these docs after https://github.com/martinvonz/jj/pull/1314 is in (feel free to write the docs for that yourself or not to do it, I'll adapt this accordingly).

Reply:
> This is an area where jj has known bugs.

Hmm, I didn't consider that. Maybe depends on how we run `jj` commands?

> I'm not sure how to better communicate the second one.

Mercurial includes a bug report instruction in exception message. I don't know the difficulty of implementing custom panic handler or something, but that's one idea.